raml-osprey-expressjs/README.md

```diff
@@ -0,0 +1,172 @@
+## Project elements
+
+*pokemon_api/package.json* - Package file with dependencies.
+*pokemon_api/api.raml* - JSON API spec that defines few routes, types and uses libraries. Available endpoints: `GET /pokemon, POST /pokemon, GET /pokemon/{id}, PATCH /pokemon/{id}, DELETE /pokemon/{id}`.
+*pokemon_api/index.js* - Osprey/Express app. Defines helper funcs/middleware; request handlers; etc. Responses have proper content-type, status codes, location header on POST; data is wrapped the way JSON API requires it;
+*pokemon_api/examples/* - Contains collection and item responses examples. These are used in a **Interacting with the server** section.
+*pokemon_api/libraries/jsonApiLibrary.raml* - Defines RAML Data Types required to satisfy JSON API requirements.
+*pokemon_api/libraries/jsonApiCollections.raml* - Defines resourceTypes and traits required to properly describe the JSON API RAML spec.
+
+## Things changed in JSON API RAML lib
```

Comment:
This section/paragraph can be moved to `libraries/README.md`

Reply:
Fixed